examples/rust/components/http-hello-world/Cargo.toml

```diff
@@ -9,4 +9,5 @@ version = "0.1.0"
 crate-type = ["cdylib"]
 
 [dependencies]
+wasmcloud-component = "0.2.0"
 wit-bindgen = "0.32"
```

Comment:
we don't need this anymore, do we?

Reply:
No not technically, but our docs do assume it already exists there. A followup in the documentation will be to add custom interfaces, which once that captures the `cargo add wit-bindgen` step I'll drop this entirely